.github/workflows/js_publish.yml

```diff
@@ -0,0 +1,79 @@
+name: Javascript wallet
```

Comment:
I'd suggest having an explicit name for the  job, something like **"Release JavaScript Wallet"** and a workflow file name like `publish_js_wallet.yml`